lib/activerecord/delay_touching.rb

```diff
@@ -87,17 +87,23 @@ def self.touch_records(attr, klass, records)
           changes[column] = current_time
           records.each do |record|
             next if record.destroyed?
+            next unless record.persisted? # A new record may have been rolled back
```

Comment:
That was one of the deficiencies I wanted to discuss.  I don't know how to identify those records.  @afiedler, do you?